packages/base-styles/_variables.scss

```diff
@@ -86,6 +86,7 @@ $widget-area-width: 700px;
  * Block & Editor UI.
  */
 
+$block-vertical-toolbar-height: $grid-unit-50;
```

Comment:
This seems quite a bit too prescriptive here. I don't think we'd use this anywhere else.

Reply:
We tend not to add variables unless we absolutely have to.

As Zoom Out is still behind an experiment I don't think we should add this.

I understand the motivation but it's a premature abstraction in my opinion.